docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/home_landing/paragraph--ncids-promo-block-internal.html.twig

```diff
@@ -50,7 +50,9 @@
   {# ###### Setup Theme ####### #}
   {% set promo_theme = paragraph.field_ncids_theme.value %}
   {% if promo_theme == 'light' %}
-      {% set promo_theme = '' %}
+    {% set promo_theme = '' %}
+  {% elseif promo_theme == 'dark' %}
```

Comment:
Can you change this to **also** match `nci-promo-block--dark`?

The reason is that on production all the dark promoblocks already have the value `nci-promo-block--dark` in the database.